src/services/user.service.js

```diff
@@ -62,6 +62,41 @@ const UserService = {
         catch(error){
             throw new Error(error);
         }
+    },
+    async getApprovedPlaces(){
+        try{
+            const response = await RequestService.get('/api/approved-places');
+            return response;
+        }
+        catch(error){
+            throw new Error(error);
+        }
+    },
+    async getRejectedPlaces(){
+        try{
+            const response = await RequestService.get('/api/rejected-places');
+            return response;
+        }
+        catch(error){
+            throw new Error(error);
+        }
+    },
+    async putRejectPlace(placeId, reason){
+        try{
+            const body = {
+                placeId: placeId,
+                reason, reason
+            }
+            const token = TokenService.getToken();
+            const headers = {
+                Authorization: token,
+                'Content-Type': 'application/json'
+            }
+            const response = await RequestService.put('/api/reject', body, headers);
+        }
+        catch(error){
+            throw new Error(error);
+        }
```

Comment:
Suggestion:
All methods for places are written in user-service file in which all methods related to users written. It is better to create separate file for those methods)

Reply:
Done)